### PR TITLE
add files for python 3.11

### PR DESCRIPTION
**Pull request by luisosmx**
Files for python 3.11 were added

### CODE REVIEW

File: 01-introduction/README.md [hunk 1]
```diff
@@ -1,108 +1,130 @@
...
 * How to use your Google account, especially with Drive
 * Some practice with Python
 
 ### Setup
-It is important to set up your virtual environment before starting with the course, please go [setup](01-introduction/setup) section to learn how to do it.
 
+It is important to set up your virtual environment before starting with the course, please go [setup](01-introduction/setup) section to learn how to do it.
 
 ## Activity
-In this activity, you will migrate and solve the package issues regarding a notebook that was created in a `Python 3.7.9` version.
```
Comment (carloslme) [inline 01-introduction/README.md:21]: This is not correct. It was created originally for 3.7.9

File: 01-introduction/README.md [hunk 2]
```diff
@@ -1,108 +1,130 @@
...
+
+In this activity, you will migrate and solve the package issues regarding a notebook that was created in a `Python 3.11.2` version.
 
 Follow the instructions below to do the activity.
+
 ### Run the existing notebook
+
 1. Clone the project `https://github.com/carloslme/intro-data-engineering.git` on your local computer.
-2. Create a virtual environment with `Python 3.7.9`
+2. Create a virtual environment with `Python 3.11.2`
```
Comment (carloslme) [inline 01-introduction/README.md:32]: This is not correct. It was created originally for 3.7.9

File: 01-introduction/README.md [hunk 3]
```diff
@@ -1,108 +1,130 @@
...
         ```
-        python3 -m venv venv
+        python -m venv venv
         ```
 
     * Activate the virtual environment
 
         ```
-        source venv/bin/activate
+        intro-data-engineering\venv\Scripts\Activate.ps1
```
Comment (carloslme) [inline 01-introduction/README.md:42]: This is for Windows, please create a section for Windows venv activation

File: 01-introduction/README.md [hunk 4]
```diff
@@ -1,108 +1,130 @@
...
         ```
 
 3. Install libraries
     Run the following command to install the other libraries.
 
     ```bash
-    pip install -r intro-data-engineering/01-introduction/activity/requirements-37.txt
+    pip install -r intro-data-engineering\01-introduction\activity\requeriments-311.txt
```
Comment (carloslme) [inline 01-introduction/README.md:49]: This is not correct. It was created originally for 3.7.9

File: 01-introduction/README.md [hunk 5]
```diff
@@ -1,108 +1,130 @@
...
+
     Verify the installation with this command:
+
     ```bash
     pip freeze
     ```
+
     Output:
     <details open>
     <summary>List of packages, click to collapse</summary>
```
Comment (carloslme) [inline 01-introduction/README.md:63]: Please create Windows section for the libraries output and keep MacOS output

File: 01-introduction/README.md [hunk 6]
```diff
@@ -1,108 +1,130 @@
...
+4. Open the `01-introduction/activity/end_to_end_machine_learning_project.ipynb` notebook and click on `Run All`.
     > **IMPORTANT!**  
-    Do not forget to select the Python 3.7.9 kernel you have already created.
+    Do not forget to select the Python 3.11.2 kernel you have already created.
 
     If everything was ok, you should be able to see the last cell with this output:
+
     ```bash
-    Predictions:	 [263527.   331884.02 221119.   ... 105722.   213199.   459125.66]
+    Predictions:[ 75446.   298423.01  82109.   ... 157926.   239522.    71491.  ]
```
Comment (carloslme) [inline 01-introduction/README.md:121]: This is not correct. It was created originally for 3.7.9

**Review by carloslme — CHANGES_REQUESTED** [review]
Please review the comments

File: 01-introduction/README.md (hunk 5 above)
Comment (luisosmx) [reply 01-introduction/README.md:63]: This was changed for python 3.11

**Review by carloslme — APPROVED** [review]
LGTM!

**Pull request merged**